notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/controller/notification/NotificationIconController.java

```diff
@@ -44,7 +44,7 @@
     private static final String ICON_PREFERENCE = "NotificationIconController.faIcon";
     private static final String ICON_DEFAULT = "fa-bell";
     private static final String ACTIVE_COLOR_PREFERENCE = "NotificationIconController.activeColor";
-    private static final String ACTIVE_COLOR_DEFAULT = "#d9534f";
+    private static final String ACTIVE_COLOR_DEFAULT = "#d50000";
```

Comment:
This rotates the color on the color scale, and sharpens the color more than the surrounding ui.
Could darkening the color work?

```js
const color = require('color');
color('#d9534f').darken(0.5).hex();
// -> #7A1D1A
```

Reply:
Hi Christian, I'm not completely sure, this change I made however does fix the contrast issue in our portal. I was able to see that #d9534f in the CSS and able to update to #d50000 and it passed our accessibility scan after that.

Reply:
got it, thanks @asgrant! :bow:

Reply:
:+1: